### PR TITLE
Update README.md.

### PR DESCRIPTION
**Pull request by henrywang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,161 +3,100 @@
 [![Build Status](https://travis-ci.org/weldr/welder-web.svg?branch=master)](https://travis-ci.org/weldr/welder-web)
 [![codecov](https://codecov.io/gh/weldr/welder-web/branch/master/graph/badge.svg)](https://codecov.io/gh/weldr/welder-web)
 
-The web interface for Composer!
+**The web interface for Composer!**
 
+Cockpit Composer is a common web interface and running as a [Cockpit](https://cockpit-project.org/) plugin. Cockpit Composer joins together with [Lorax Composer](https://github.com/weldr/lorax/tree/lorax-composer) to generate custom images suitable for deploying systems, or as images ready to upload the cloud.
```
Comment (martinpitt) [inline README.md]: "common web interface" does not really explain anything. How about explaining what it is first, and then how it  works? There are also some words missing. How about

"Composer generates custom images suitable for deploying systems or uploading to the cloud. It integrates into Cockpit as a frontend for Lorax  Composer."

File: README.md [hunk 2]
```diff
@@ -3,161 +3,100 @@
...
-`jest` provides its own instrumentation and it will break if we do double instrumentation.
-Disabled by default!
+* First install Cockpit on your local machine as described in: https://cockpit-project.org/running.html.
+* Next install and start lorax-composer:
+```
+    $ sudo yum install lorax-compose
+    $ sudo systemctl start lorax-composer
+```
 
+Cockpit Composer uses Node.js during development. Node.js is not used at runtime. To make changes on Cockpit you'll want to install Node.js, NPM.
```
Comment (martinpitt) [inline README.md]: This sentence would fit better in the next enumeration point, where it avoids the duplication.

File: README.md [hunk 3]
```diff
@@ -3,161 +3,100 @@
...
-&nbsp; &nbsp; ✓ Hot Module Replacement ([HMR](https://webpack.github.io/docs/hot-module-replacement.html)) /w [React Hot Loader](http://gaearon.github.io/react-hot-loader/)<br>
-&nbsp; &nbsp; ✓ Cross-device testing with [Browsersync](https://browsersync.io/) (see [`run.js#start`](run.js))<br>
+This will cause cockpit to read JavaScript and HTML files directly from the built package output directory instead of using the installed Cockpit UI files.
 
-### Directory Layout
+Now you can log into Cockpit Composer on your local Linux machine at the following address. Use the same user and password that you used to log into your Linux desktop.
 
-```shell
-.
-├── /components/                # Shared or generic UI components
```
Comment (martinpitt) [inline README.md]: The layout may have been slightly out of date, but it's still useful to get a summary of the file organization. Can we keep this?

**Comment by henrywang** [comment]
Thanks @martinpitt. I'll bring layout back and make a update when #497 landed.

**Comment by jgiardino** [comment]
Thanks for making these updates, @henrywang!

Just FYI, my build process is slightly different, since I'm working with Cockpit in a VM, so I can't verify that this outline is accurate.

**Pull request merged**